deployment/terraform/participant/outputs.tf

```diff
@@ -6,6 +6,20 @@ output "assets_storage_account" {
   value = azurerm_storage_account.assets.name
 }
 
+output "assets_storage_account_key" {
+  value = azurerm_storage_account.assets.primary_access_key
+  sensitive = true
+}
+
+output "inbox_storage_account" {
+  value = azurerm_storage_account.inbox.name
+}
+
+output "inbox_storage_account_key" {
+  value = azurerm_storage_account.inbox.primary_access_key
+  sensitive = true
+}
+
```

Comment:
@algattik as storage key is available as output.So I think we can simplify system-test here https://github.com/eclipse-dataspaceconnector/MinimumViableDataspace/blob/dff0b11c937cca2975e3be6c949c56051443fec8/system-tests/src/test/java/org/eclipse/dataspaceconnector/system/tests/local/BlobTransferIntegrationTest.java#L76 , as we do not need to traverse all keys from KV to see which one is for the storage account.

What do you think ? I can add this in separate story and cover it later if we agree.

Reply:
The reason this is done in system test code is to make setup easier for local development.